src/caret_analyze/record/records_service/response_time.py

```diff
@@ -759,6 +745,42 @@ def _to_timeseries(self, input_column, output_column):
 
         return t_in, latency
 
+    def to_best_case_records(self) -> RecordsInterface:
+        records = self._records.to_range_records()
+        input_column = records.columns[1]
+        output_column = records.columns[-1]
+        return self._to_records(input_column, output_column)
+
+    def to_worst_case_records(self) -> RecordsInterface:
+        records = self._records.to_range_records()
+        input_column = records.columns[0]
+        output_column = records.columns[-1]
+        return self._to_records(input_column, output_column)
+
```

Comment:
@rokamu623 
I think that using` to_range_records` in best_case and worst_case are not best way just for column name.
Although there is no way to get column names in ResponseTime class, it is better that, for example, create a new variable `self._input_records = records` in the ResponseTime class and then write as `input_column = f'{self._input_records.columns[0]}_min or max'.`
However, it is a little more complicated to write than before.